### ISSUE TITLE
[4.0] [a11y] phpinfo in system information is not accessible

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#23154]
### Steps to reproduce the issue
Have a look in the sourcecode of the php info. The phpinfo is delivered by php in form of html-tables.
There are several tables with headers in-between.

### Actual result
The php info tables are not accessible, no caption, no scope.

**Comment by mbabker** [comment]
This is a can't fix.  The PHP info tab is the result of the core `phpinfo()` function with some regexes to make the tables use the template's styling.  Changing the markup means creating our own function and trying to extract all the information that is given in other ways, or someone writing a patch for core PHP changing the returned markup structure.

**Comment by chmst** [comment]
Maybe we could expand the regexes a little bit, such avoiding empty title headers and adding the scope?

**Comment by mbabker** [comment]
Personally, I would not suggest manipulating the output of `phpinfo()` beyond "simple" changes that can be easily applied.  Trying to change specific nodes is asking for trouble.

Your best bet is to see if PHP itself will make adjustments to the generated markup.  Anything else is high risk and prone to breaking.

**Comment by brianteeman** [comment]
As Michael says this can't be fixed and we should not attempt to manipulate external code. All you can do is to raise a bug with php

Should be closed as cant fix
